fud2/fud-core/src/cli.rs

```diff
@@ -175,6 +175,10 @@ struct FakeArgs {
     /// planner for the backend
     #[argh(option, default = "Planner::Legacy")]
     planner: Planner,
+
+    /// print commands which would be run instead of running them
+    #[argh(switch)]
+    print_commands: bool,
```

Comment:
Maybe this would make more sense as a mode (`-m`) option, instead of a dedicated flag? I don't think it makes sense to combine with modes other than `run`, for instance, because that's the only one that actually invokes Ninja. And it's kind of a sibling to `-m plan`, which prints out the fud2 view of the plan (and this shell-ish view of the plan).

So maybe `-m cmds` or whatever would make marginally more sense?